workshop/gcloud-compute/porter.yaml

```diff
@@ -1,7 +1,7 @@
 name: porter-gcloud-compute
```

Comment:
maybe call this `name: gcloud-compute` just to match the change you have been making to remove the extra porter- prefix on our bundles?